tests/app/dao/test_jobs_dao.py

```diff
@@ -246,8 +247,15 @@ def test_get_scheduled_jobs_gets_ignores_jobs_not_scheduled(notify_db, notify_db
     assert jobs[0].id == job_scheduled.id
 
 
-def test_get_scheduled_jobs_gets_ignores_jobs_scheduled_in_the_future(notify_db, notify_db_session):
-    one_minute_in_the_future = datetime.utcnow() + timedelta(minutes=1)
-    sample_job(notify_db, notify_db_session, scheduled_for=one_minute_in_the_future, job_status='scheduled')
+def test_get_scheduled_jobs_gets_ignores_jobs_scheduled_in_the_future(
```

Comment:
two things:

1) you don't need to include notify_db and notify_db_session as fixture params, since they're prerequisites of sample_scheduled_job they'll be loaded anyway

2) does it make sense to add a new test that tests the inverse of this? ie:
```
def test_get_scheduled_jobs_returns_jobs_scheduled_in_the_past(sample_scheduled_job):
    with freeze_time(datetime.now() + timedelta(hours=2)):
        jobs = dao_get_scheduled_jobs()
    assert len(jobs) == 1
```